app/assets/stylesheets/spam2.css

```diff
@@ -0,0 +1,124 @@
+  .nav_top  {
```

Comment:
Please format this whole file.

Reply:
Done!
Thanks!

Reply:
There are still some serious formatting errors. Don't do this manually, use a tool.

Reply:
I changed it and used a tool, is it fine now?